src/commcare_cloud/ansible/roles/couchdb2-preinstall/tasks/main.yml

```diff
@@ -31,8 +31,8 @@
 
 - name: Add Couch apt repo (for libmozjs dependency)
   block:
-    - apt_key: url=https://couchdb.apache.org/repo/bintray-pubkey.asc state=present
-    - apt_repository: repo="deb https://apache.bintray.com/couchdb-deb bionic main" state=present
```

Comment:
@skodde rather than having the process be that we manually remove these, what do you think about adding these two lines back but changing `state=present` to `state=absent`? I think that would be the normal process for an update like this and would make it so we don't have to manually remove files from each machine

Reply:
Fixed here: https://github.com/dimagi/commcare-cloud/pull/4878/commits/be095f19d2dc4180079da3d780022e8d028dc50e